src/FilenameParsing/FileIDHelperResolutionStrategy.php

```diff
@@ -363,52 +389,78 @@ public function buildFileID($tuple)
         );
     }
 
+    /**
+     * todo: needs documenting
```

Comment:
Document

Reply:
Those methods that didn't have a PHPDoc block are implemented from an interface that has the doc block. The interface doc block are inherited.